Extension/CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Version 0.26.2-insider: November 11, 2019
+
+### Enhancements
+* Reworked how a source file is selected for TU creation when opening a header file. [#2856](https://github.com/microsoft/vscode-cpptools/issues/2856)
+* Reset memory usage of IntelliSense process if it grows beyond a threshold. [#4119](https://github.com/microsoft/vscode-cpptools/issues/4119)
+
+### Bug Fixes
+* Fix child process creation when the Windows code page is set to a language with non-ASCII characters and there are non-ASCII characters in the extension's install path. [#1560](https://github.com/microsoft/vscode-cpptools/issues/1560)
+* Fix path canonicalization of UNC paths to avoid duplicate files opening with different casing. [#2528](https://github.com/microsoft/vscode-cpptools/issues/2528), [#3980](https://github.com/microsoft/vscode-cpptools/issues/3980)
+* Fix an issue with Rename UI that could cause the rename to not be applied. [#4510](https://github.com/microsoft/vscode-cpptools/issues/4510), [#4504](https://github.com/microsoft/vscode-cpptools/issues/4504)
```

Comment:
It looks like #4510 belongs to the next issue below. The `[#4517]` could be removed or changed to `[PR #4517]`.